### DISCUSSION TITLE
STUDENT DEVELOPER PACK

### DISCUSSION THREAD

**Discussion by KHIMBRAINS** [community/community#79211] [GitHub Education]
i need to be approved for student developer pack please

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @KHIMBRAINS , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category and are explicit about what your question entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!

**Comment by a7medelbatal** [comment]
i don't understand you
